backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="overriding-kopia-algorithms"]
+= Overriding Kopia hashing, encryption, and splitter algorithms
+include::_attributes/common-attributes.adoc[]
+:context: overriding-kopia-algorithms
+:overriding-kopia-algorithms:
```

Comment:
Just wondering what is the purpose of this variable (overriding-kopia-algorithms)?

Reply:
I have removed the variable as the module does not have any if-def statements. Thanks.